### DISCUSSION TITLE
Starting a Dedicated server for my small community

### DISCUSSION THREAD

**Discussion by CyberViking07** [Vdauphin/HeartsAndMinds#1628] [Question and answer]
Currently we are using KP liberation and love ace. We are wondering what we can change what addons do you recommend? Can we turn off the ticket system? it might be punishing for small group of like 5-10 of us. we would like to try out H&M but worried it will be to much.

**Comment by Vdauphin** [comment]
Hello,

> We are wondering what we can change what addons do you recommend?

CBA, ACE and ACRE will give you the best experience

> Can we turn off the ticket system?

Yes, check mission parameters. But I would advise to keep all gameplay. Instead, you should reduce difficulty of AI or number of enemies/IED/...

Cheers

**Comment by crispopp** [comment]
You can turn off the ticket system in KP Liberation by going to the admin settings and disabling it, which can simplify management for a small group of 5-10 people. For addons, consider performance enhancers like cache plugins or security plugins. If you want to try H&M, start with its basic features to avoid overwhelm.  **[Capra Host](https://caprahost.com/)** can be a reliable option for support and additional features.
